drivers/docker/driver.go

```diff
@@ -1004,6 +1004,15 @@ func (d *Driver) StopTask(taskID string, timeout time.Duration, signal string) e
 		return fmt.Errorf("failed to parse signal: %v", err)
 	}
 
+	defer func(h *taskHandle) {
+		h.dloggerPluginClient.Kill()
```

Comment:
What do you think of deferring this? If we kill the plugin we can't reliably call the Stop RPC.